NativeApp/src/screens/Booking.js

```diff
@@ -2,12 +2,20 @@ import React, { Component } from 'react';
 import { PropTypes as PT } from 'prop-types';
 import { Icon } from 'react-native-elements';
 import Booking from '../components/Booking';
+import { WHITE } from '../styles/colors';
+import { H2_SIZE } from '../styles/text';
 
 export default class BookingScreen extends Component {
   static navigationOptions = {
     title: 'Booking',
     headerBackTitle: null,
-    headerBackImage: () => <Icon type="font-awesome" name="arrow-left" size={20} color="grey" />,
+    headerTitleStyle: {
+      color: WHITE,
+      marginHorizontal: 0,
+      fontSize: H2_SIZE,
+      fontFamily: 'oswaldRegular',
+    },
+    headerLeft: <Icon containerStyle={{ paddingLeft: 15 }} type="font-awesome" name="arrow-left" size={20} color="white" />,
```

Comment:
This would be more readable with the props split onto multiple lines
```
headerLeft: 
  <Icon 
    containerStyle={{ paddingLeft: 15 }} 
    type="font-awesome" 
    name="arrow-left" 
    size={20} 
    color="white" 
  />,
```
or whatever compiles lol

Reply:
I have put it back to headerBackImage as the back button does not work without it